packages/resourceful-redux/src/request-statuses-plugin/delete.js

```diff
@@ -76,35 +76,28 @@ function delSucceed(state, action, {initialResourceMeta}) {
 
   const hasIds = idList && idList.length;
 
-  // If we have no, IDs, list, nor request label, then there is nothing to update
-  if (!hasIds && !label) {
+  // If we have no IDs, then there is nothing to update
```

Comment:
I think we accidentally deleted part of this comment. Can we go back to the old comment but with `request label` swapped with `request name`?

Reply:
Yea, it originally was not correct (included `list` but not in the logic) and then when I took out the `label` check I didn't add it back in for `request`. Whoops.